planning/obstacle_cruise_planner/src/planner_interface.cpp

```diff
@@ -401,12 +373,13 @@ std::vector<TrajectoryPoint> PlannerInterface::generateStopTrajectory(
 
   stop_planning_debug_info_.set(
     StopPlanningDebugInfo::TYPE::STOP_CURRENT_OBSTACLE_DISTANCE,
-    closest_obstacle_dist - abs_ego_offset);  // TODO(murooka)
+    closest_stop_obstacle->dist_to_collide);  // TODO(murooka)
```

Comment:
Is this correct? Seems `dist_to_collide` does not consider the following distances
- the distance from the path start point and the ego position
- the distance from the base link to the ego front position.